integr8ly/integreatly-operator
# add cro 0.22.1 (release-v2.7)

bump cro to 0.22.1

verification:
- ensure rhmi installs successfully
- ensure cro is 0.22.1 in both the rhmi status block and the container image
